{
	"id": "AVG-1135",
	"advisories": [
		{
			"id": "AVG-1135",
			"severity": [
				{
					"type": "vendor",
					"source": "security.archlinux.org",
					"vendor": "Medium"
				}
			],
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/AVG-1135"
				}
			]
		},
		{
			"id": "ASA-202004-16",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/ASA-202004-16"
				}
			]
		},
		{
			"id": "ASA-202004-16",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/ASA-202004-16"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"id": "CVE-2020-11810",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2020-11810"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "arch",
			"vulnerable": true,
			"package": {
				"name": "openvpn"
			},
			"affected": {
				"type": "pacman",
				"range": [
					{
						"lt": "2.4.9-1"
					}
				],
				"fixed": [
					"2.4.9-1"
				]
			}
		}
	]
}
